sound/snapcast/files/snapclient.init

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh /etc/rc.common
+# Author: Johannes Pohl <johannes.pohl@badaix.de>
+START=90
+
+USE_PROCD=1
+
+NAME=snapclient
+PROG=/usr/bin/$NAME
+PID_FILE=/var/run/$NAME.pid
+CONFIG_FILE=/etc/default/$NAME
```

Comment:
Please convert it to use UCI config. You may take [v2ray-core](https://github.com/openwrt/packages/tree/master/net/v2ray-core/files) as an example.

Reply:
@xabolcs To get started with / finish the PR, UCI config can offer a single option to a native snapserver or snapclient config file like in the example:

```
config snapserver 'config'
	option conf '/etc/default/snapserver'
```

As time passes, common uses cases could get better integration via new UCI options like stream sources, etc

Reply:
@davidandreoletti , I think that won't do it: I mean the review is about to avoid some new config parsing logic as there is already one.
I'm already rebasing my branch and implementing this, just have no time to finish. :see_no_evil: 
So I have some work-in-progress commit, but the rebase itself isn't finished yet, so I can't publish it.

Please be patient, I will try to find time to finish this.

Reply:
@xabolcs Hi, where you able to find some time eventually ?

Reply:
Addressed comment about UCI config! Thanks again for the pointer! :pray: